libs/gl-testing/tests/test_node.py

```diff
@@ -22,8 +22,7 @@ def test_node_start(scheduler, clients):
 
 
 def test_node_connect(scheduler, clients, bitcoind):
-    """Register and schedule a node, then connect to it.
```

Comment:
Feels like the changes here should have been a separate format commit. Are there any logic changes in here?